test/PipeBlocking.ll

```diff
@@ -0,0 +1,72 @@
+; RUN: llvm-as %s -o %t.bc
+; RUN: llvm-spirv %t.bc -spirv-text -o %t.spt
+; RUN: FileCheck < %t.spt %s --check-prefix=CHECK-SPIRV
+; RUN: llvm-spirv %t.bc -o %t.spv
+; RUN: llvm-spirv -r %t.spv -o %t.bc
+; RUN: llvm-dis < %t.bc | FileCheck %s --check-prefix=CHECK-LLVM
+
+; ModuleID = 'test/CodeGenOpenCL/pipe_builtin.cl'
+source_filename = "test/CodeGenOpenCL/pipe_builtin.cl"
+target datalayout = "e-i64:64-v16:16-v24:32-v32:32-v48:64-v96:128-v192:256-v256:256-v512:512-v1024:1024"
+target triple = "spir64-unknown-unknown"
+
+%opencl.pipe_ro_t = type opaque
+%opencl.pipe_wo_t = type opaque
+
+; CHECK-SPIRV: 2 Capability BlockingPipesINTEL
+; CHECK-SPIRV: 8 Extension "SPV_INTEL_blocking_pipes"
+; CHECK-SPIRV: 3 TypePipe {{[0-9]+}} 0
+; CHECK-SPIRV: 3 TypePipe {{[0-9]+}} 1
+; CHECK-SPIRV: 7 ReadPipeBlockingINTEL {{[0-9]+}} {{[0-9]+}} {{[0-9]+}} {{[0-9]+}} {{[0-9]+}} {{[0-9]+}}
```

Comment:
According to the extension specification these instructions must have 5 operands.

Reply:
Thanks! Done